app/services/sources/sources.ts

```diff
@@ -241,7 +240,7 @@ export class SourcesService extends StatefulService<ISourcesState> {
      * otherwise OBS thinks it's still attached
      * and won't release it. */
     if (source.channel !== void 0) {
-      obs.Global.setOutputSource(source.channel, null);
+      obs.Global.setOutputSource(source.channel, (null as unknown) as obs.ISource);
```

Comment:
We probably need to update the type definitions inside OSN for this one since `null` is a valid value.  No action here, this is fine for now.